### DISCUSSION TITLE
Does anybody use spark .net in production yet?

### DISCUSSION THREAD

**Discussion by apostarn** [dotnet/spark#809] [Q&A]
Hi,

Does anybody use spark .net in production yet and if yes, what are the use cases?
I'm thinking of using it in financial sector to parallelize the calculation of various risks for tradeable assets. 
Would like to get an idea if there is anybody out there who already done approximately the same in production and how successful the setup is.

**Comment by MikeRys** [comment]
Hi Andrey

We have several Microsoft teams and other customers using .NET for Spark. We have some customer testimonials in our v1.0 announcement at https://techcommunity.microsoft.com/t5/azure-synapse-analytics/microsoft-and-the-net-foundation-announce-the-release-of-version/ba-p/1820908. Please note that a lot of production customers are probably not here on Github, since many of them are using it through their HDInsight or Azure Synapse Spark experiences.

If you have any further questions on how to use it, feel free to reach out. What Spark engine/platform are you looking at using?

Cheers
Michael

**Reply by dbeavon** [reply]
**Please note that a lot of production customers are probably not here on Github, since many of them are using it through their HDInsight or Azure Synapse Spark experiences.**

Can you tell me if you are aware of anyone using this in production within Azure Databricks?  Prior to the 1.0 GA release, there were warnings in the docs about the fact that .Net for Spark were unsupported in production.  Now it appears the warnings are all removed , see:
https://docs.microsoft.com/en-us/dotnet/spark/tutorials/databricks-deployment

... So I'm assuming that the folks at Azure Databricks are now ready to support this?  (... of course I would probably come ask about my problems in the github community as well ... but sometimes managers like to know that there was a Microsoft support ticket that was created, for whatever it's worth).
